library/Notifications/Widget/Timeline/Entry.php

```diff
@@ -33,6 +35,21 @@ public function getColor(int $transparency): string
         return TimeGrid\Util::calculateEntryColor($this->getMember()->getName(), $transparency);
     }
 
+    /** Sets Content of a Flyout that is shown when the Entry is hovered
+     * @param ValidHtml $content
+     * @return $this
+     */
```

Comment:
* Make always a line break after `/**`
* Please insert empty lines after the subject…
* …and after the parameter list

e.g.

```
/**
 * subject
 *
 * @param …
 *
 * @return …
 */
```